src/cxristian/Matriz_Cuadrada.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+
+public class Matriz_Cuadrada {
```

Comment:
I've noticed that you are mixing english with spanish for naming. Let's rename spanish names and use english for all.